jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -717,6 +717,12 @@
         jenkins-timeout: 170
         timeout: 70
         trigger-job: 'ci-kubernetes-build'
+    - kubernetes-e2e-cos-docker-validation:
+        frequency: 'H/5 * * * *' # At least every 30m
```

Comment:
Note that this is going to run the same tests with the same image over and over again. Once every 30 min is fine. We just need to remember to stop it once we have got enough passes.